File: python/src/acp_factory/providers/copilot_cli.py
"""
GitHub Copilot CLI agent configuration
"""

from acp_factory.types import AgentConfig

# Default configuration for GitHub Copilot CLI agent via --acp flag
copilot_cli_config = AgentConfig(
    command="npx",
    args=["@github/copilot", "--acp"],
    env={},
)

File: src/providers/copilot-cli.ts
/**
 * GitHub Copilot CLI agent configuration
 */

import type { AgentConfig } from "../types.js";

/**
 * Default configuration for GitHub Copilot CLI agent via --acp flag
 */
export const copilotCliConfig: AgentConfig = {
  command: "npx",
  args: ["@github/copilot", "--acp"],
  env: {},
};
